content/docs/for-platform-operators/concepts/build-run-images.md

```diff
@@ -0,0 +1,94 @@
+* The image config's `Env` field has the environment variable `PATH` set to a valid set of paths or explicitly set to empty (`PATH=`)
+
+The platform should ensure that:
+
+* The image config's `Label` field has the label `io.buildpacks.base.maintainer` set to the name of the image maintainer.
+* The image config's `Label` field has the label `io.buildpacks.base.homepage` set to the homepage of the image.
+* The image config's `Label` field has the label `io.buildpacks.base.released` set to the release date of the image.
+* The image config's `Label` field has the label `io.buildpacks.base.description` set to the description of the image.
+* The image config's `Label` field has the label `io.buildpacks.base.metadata` set to additional metadata related to the image.
+
+#### Anatomy of a build image
+
+Typically, a `build` image may include:
+
+* Shell
+* C-compiler
+* Minimal operating system distribution, such as Linux utilities that build systems might call out to
+* Build time libraries
+
+### Run image
+
+A `run image` is an `OCI image` that serves as the base image for the final application image.
+
+A typical run image might determine:
+
+* The OS distro or distroless OS in the launch environment
+* OS packages installed in the launch environment
+* Trusted CA certificates in the launch environment
+* The default user in the run environment
+
+The platform must ensure that:
+
+* The image config's `Env` field has the environment variable `PATH` set to a valid set of paths or explicitly set to empty (`PATH=`)
+
+The platform should ensure that:
+
+* The image config's `User` field is set to a user with a **DIFFERENT** user [UID/SID](https://github.com/buildpacks/spec/blob/main/README.md#operating-system-conventions) as the build image
+* The image config's `Label` field has the label `io.buildpacks.base.maintainer` set to the name of the image maintainer
+* The image config's `Label` field has the label `io.buildpacks.base.homepage` set to the homepage of the image
+* The image config's `Label` field has the label `io.buildpacks.base.released` set to the release date of the image.
+* The image config's `Label` field has the label `io.buildpacks.base.description` set to the description of the image
+* The image config's `Label` field has the label `io.buildpacks.base.metadata` set to additional metadata related to the image
+* The image config's `Label` field has the label `io.buildpacks.rebasable` set to `true` to indicate that new run image versions maintain [ABI-compatibility](https://en.wikipedia.org/wiki/Application_binary_interface) with previous versions (see [Compatibility Guarantees](https://github.com/buildpacks/spec/blob/main/platform.md#compatibility-guarantees)).
+
+#### Anatomy of a run image
+
+A `runtime` image may contain:
```

Comment:
What's the difference between the "run image" and a "runtime" image?

Reply:
I think nothing... I think we decided to use "runtime base image" as opposed to "run image" because the latter was more of a CNB construct whereas the former would be recognized more widely.